File: src/main/java/com/backend/app/shared/models/entities/Customer.java
package com.backend.app.shared.models.entities;

import java.util.Date;

import com.fasterxml.jackson.annotation.JsonProperty;

import jakarta.annotation.Nullable;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "customers")
public class Customer {
    
    @Id
    @Column(name = "id")
    private String id;
    
    @Column(name = "firstname")
    private String firstName;
    
    @Column(name = "lastname")
    @Nullable
    private String lastName;

    @Column(name = "phone_one")
    @Nullable
    private String phoneOne;

    @Column(name = "phone_two")
    @Nullable
    private String phoneTwo;

    @Column(name = "created_date")
    private Date createdDate;

    @Column(name = "created_by")
    private String createdBy;

    @Column(name = "updated_date")
    private Date updatedDate;

    @Column(name = "updated_by")
    private String updatedBy;
}

File: user-svc/UserService/Models/Entities/Customer.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace UserService.Models.Entities
{
    [Table("customers")]
    public class Customer
    {
        [Key]
        [Column("id")]
        public required string Id { get; set; }

        [Column("firstname")]
        public required string FirstName { get; set; }

        [Column("lastname")]
        public string? LastName { get; set; }

        [Column("phone_one")]
        public string? PhoneOne { get; set; }

        [Column("phone_two")]
        public string? PhoneTwo { get; set; }

        [Column("created_date")]
        public required DateTime CreatedDate { get; set; }

        [Column("created_by")]
        public required string CreatedBy { get; set; }

        [Column("updated_date")]
        public required DateTime UpdatedDate { get; set; }

        [Column("updated_by")]
        public required string UpdatedBy { get; set; }
    }
}
